src/sync.h

```diff
@@ -14,6 +15,14 @@
 namespace librealsense
 {
 
+    typedef int stream_id;
+    inline std::string generate_matcher_name(rs2_stream stream_type, stream_id stream)
```

Comment:
The function seem to be redundant as it is just a wrapper for rs2_stream_to_string(... plus a whitespace